11.0.Dockerfile

```
@@ -188,8 +188,8 @@ ONBUILD ARG UID=1000
 ONBUILD ARG GID=1000
 
 # Enable Odoo user and filestore
-ONBUILD RUN groupadd -g $GID odoo \
-    && useradd -md /home/odoo -s /bin/false -u $UID -g $GID odoo \
+ONBUILD RUN groupadd -g $GID odoo || true
```

Do this:

```suggestion
ONBUILD RUN (groupadd -g $GID odoo || true) \
```

... and let the next line untouched, please.

Also do the same for all other dockerfiles.

---

I have updated my branch.